### PR TITLE
Update activities endpoint with programFspConfigName

### PR DESCRIPTION
**Pull request by RubenGeo**
[AB#31245](https://dev.azure.com/redcrossnl/7d17f2a9-3f18-40f1-8c29-98568fbca7e4/_workitems/edit/31245) 

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [ ] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

### CODE REVIEW

**Comment by jannisvisser** [comment]
@RubenGeo tests are failing still

**Comment by RubenGeo** [comment]
Yes, the portalicious tests are broken since one of the rebases. @aberonni Wants to fix them we we are nearing the merge of this branch. So out of scope for this PR

File: interfaces/Portalicious/src/app/pages/project-registration-activity-log/components/activity-log-expanded-row/activity-log-expanded-row.component.ts [hunk 1]
```diff
@@ -113,7 +113,7 @@ export class ActivityLogExpandedRowComponent
           },
           {
             label: $localize`FSP`,
-            value: item.attributes.fsp,
+            value: item.attributes.financialServiceProviderConfigurationLabel,
```
Comment (jannisvisser) [inline interfaces/Portalicious/src/app/pages/project-registration-activity-log/components/activity-log-expanded-row/activity-log-expanded-row.component.ts:116]: is this fine? financialServiceProviderConfigurationLabel is a localizedString, whereas 'fsp' was not right?

**Review by jannisvisser — CHANGES_REQUESTED** [review]

File: interfaces/Portalicious/src/app/pages/project-registration-activity-log/components/activity-log-expanded-row/activity-log-expanded-row.component.ts (hunk 1 above)
Comment (RubenGeo) [reply interfaces/Portalicious/src/app/pages/project-registration-activity-log/components/activity-log-expanded-row/activity-log-expanded-row.component.ts]: You are right :). 

The PR of dom is now merged in [feat.refactor-registration-data](https://github.com/global-121/121-platform/tree/feat.refactor-registration-data). So rebased this branch, so I could actually test the functionality, which now works

File: interfaces/Portalicious/src/app/pages/project-registration-activity-log/components/activity-log-expanded-row/activity-log-expanded-row.component.ts [hunk 2]
```diff
@@ -113,7 +116,9 @@ export class ActivityLogExpandedRowComponent
           },
           {
             label: $localize`FSP`,
-            value: item.attributes.fsp,
+            value: this.translatableStringService.translate(
+              item.attributes.financialServiceProviderConfigurationLabel,
+            ),
```
Comment (aberonni) [inline interfaces/Portalicious/src/app/pages/project-registration-activity-log/components/activity-log-expanded-row/activity-log-expanded-row.component.ts:121]:
```suggestion
            value: item.attributes.financialServiceProviderConfigurationLabel,
```

IIRC you don't need to translate. If you pass in a translatable string, the underlying component will try to translate it. (Needs to be tested).

**Pull request merged**